_data/sites.json

```diff
@@ -4225,6 +4225,17 @@
         ]
     },
 
+    {
+        "name": "Mail.ru",
+        "url": "https://account.mail.ru/user/delete",
+        "difficulty": "impossible",
+        "notes": "You can deactivate your account and delete all data, but not entirely delete the account itself.",
```

Comment:
It might be worth adding a link to that support document as a reference for this note.